R/build_lm.R

```diff
@@ -1006,6 +1086,15 @@ tidy.glm_exploratory <- function(x, type = "coefficients", pretty.name = FALSE,
         dplyr::ungroup()
       ret
     },
+    vif = {
+      if (!is.null(x$vif) && "error" %nin% class(x$vif)) {
+        ret <- vif_to_dataframe(x)
+      }
+      else {
+        ret <- data.frame() # Skip output for this group. TODO: Report error in some way.
```

Comment:
Should describe that it returns an empty data frame in case of error at the beginning of the function.

Reply:
Added comment.